contents/handbook/growth/marketing/index.md

```diff
@@ -76,71 +74,71 @@ c) We didn't communicate the benefits of the product well enough
  
 We don't believe companies will be long term customers of a competitor because they did a better job of spamming them with generic content. We know this because we frequently have customers switching from a competitor to us - they are not afraid to do this.
  
-Addressing a) is the responsibility of everyone at PostHog. The marketing team's specific job is to avoid spending time advertising to people in group b), and making sure we do a great job avoiding c). This means:
+Tackling a) is the responsibility of everyone at PostHog. The marketing team's specific job is to avoid spending time advertising to people in group b), and making sure we do a great job avoiding c). This means:
  
 - Making sure our comms are extremely high quality
-- Sharing our messages in the right places, where relevant customers can see them
+- Sharing our messages in the right places, where relevant users can see them
 - Spending enough time and/or money in those places so that our messages get through
 
-### No sneaky shit
+### No sneaky stuff
```

Comment:
There's no need to censor ourselves.

Reply:
FYI I changed it to `shenanigans` mainly as a joke, because @charlescook-ph said that was the biggest change. I don't _really_ have strong opinions on this. I

Reply:
I just thought it was a bit 'try hard tech bro' on reflection. I don't feel strongly about it, so happy to leave it as is!